install/helm/gloo/templates/1-gloo-deployment.yaml

```
@@ -109,6 +117,16 @@ spec:
           {{- if not .Values.gloo.deployment.floatingUserId }}
           runAsUser: {{ printf "%.0f" (float64 .Values.gloo.deployment.runAsUser)}}
           {{- end }}
+        ports:
+        - containerPort: {{ .Values.gloo.deployment.validationPort }}
+          name: probe
```

i know we use it for the probe but the port name should probably be a bit more descriptive about the service behind it
```suggestion
          name: validation
```